testing/EstNumJellies.py

```diff
@@ -94,7 +113,15 @@ def compute_Njelly_pink_est(self):
                   +"computing estimate.\n"
 
         # NE24: What other checks might be useful? What is a better way to do this?
-
+        # We might want to check if the final int(n) is an integer. 
+        # If it isn't, it should be rounded to one. This can be checked with the assert type(n) code. 
+        # I would think that's all the checks we need. We need all the values to be the right type and range.
+        
+        # I'm not sure what a better way would be to run this code. It also doesn't do anything useful...
+        # I'm not entirely sure what self is. Is it an array? A class? It doesn't seem all that necessary.
```

Comment:
self is python's way of doing object-oriented stuff. If you take it away it won't work. Self stores all of the things associated with its class.